File: MoonbeltPartyManagerApp/src/app/models/person.ts
import { jsonObject, jsonMember } from '../../../node_modules/typedjson';

@jsonObject
export class Person {
    @jsonMember({ name: "id", constructor: Number })
    id!: number;
    
    @jsonMember({ name: "firstName", constructor: String })
    firstName!: string;

    @jsonMember({ name: "lastName", constructor: String })
    lastName!: string;

    @jsonMember({ name: "email", constructor: String })
    email!: string;

    @jsonMember({ name: "phoneNumber", constructor: String })
    phoneNumber!: string;
 
    public constructor(init?: Partial<Person>)
    {
       Object.assign(this,init);
    }
}

File: MoonbeltEventManager/Models/Person.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace MoonbeltEventManager.Models
{
    public partial class Person
    {
        public Person()
        {
            PersonParties = new HashSet<PersonParty>();
        }

        public int Id { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string Email { get; set; }
        public string PhoneNumber { get; set; }

        public virtual ICollection<PersonParty> PersonParties { get; set; }
    }
}
